meteor/lib/collections/PartInstances.ts

```diff
@@ -146,7 +146,7 @@ export function findPartInstanceInMapOrWrapToTemporary<T extends Partial<PartIns
 }
 
 export function findPartInstanceOrWrapToTemporary<T extends Partial<PartInstance>>(
-	partInstances: { [partId: string]: T | undefined },
+	partInstances: { [partId: string]: T | undefined } | Map<PartId, T>,
```

Comment:
Wouldn't it be better to go all the way here? `partInstances: Map<PartId, T>,` ?

Reply:
It should stay as it was before because there's another function that handles `Map<PartId, T>`